engine/servermaster/jobop/backoff.go

```diff
@@ -92,11 +82,12 @@ type backoffEvent struct {
 
 // NewJobBackoff creates a new job backoff
 func NewJobBackoff(jobID string, options ...BackoffOption) *JobBackoff {
+	defaultConfig := NewDefaultBackoffConfig()
```

Comment:
It seems `config` and `BackoffOption` have the same effect. Can we just remove the `BackoffOption`?

Reply:
Removed `BackoffOption`, PTAL.